gstreamer-plugin/git-patch/0001-svthevcenc-Add-svthevc-encoder-element.patch

```diff
@@ -0,0 +1,2123 @@
++          0, 2, PROP_TUNE_DEFAULT, G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_LATENCY_MODE,
++      g_param_spec_uint ("latency-mode", "Latency Mode",
++          "Flag to enable for lower latency mode: 0=Normal Latency, 1=Low Latency",
++          0, 1, PROP_LATENCY_MODE_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_BASE_LAYER_SWITCH_MODE,
++      g_param_spec_uint ("baselayer-mode", "Base Layer Switch Mode",
++          "Random Access Prediction Structure type setting: 0=Use B-frames in the base layer pointing to the same past picture, 1=Use P-frames in the base layer",
++          0, 1, PROP_BASE_LAYER_SWITCH_MODE_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_BITRATE,
++      g_param_spec_uint ("bitrate", "Bitrate",
++          "Bitrate in kbit/sec",
++          1, G_MAXINT, PROP_BITRATE_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_KEY_INT_MAX,
++      g_param_spec_int ("key-int-max", "Key-frame maximal interval",
++          "Distance Between Intra Frame inserted: -1=no intra update. -2=auto",
++          -2, 255, PROP_KEY_INT_MAX_DEFAULT, G_PARAM_READWRITE));
++
++  g_object_class_install_property (gobject_class, PROP_INTRA_REFRESH,
++      g_param_spec_uint ("intra-refresh", "Intra Refresh",
++          "1=CRA (Open GOP), 2=IDR (Closed GOP)",
++          1, 2, PROP_INTRA_REFRESH_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_CORES,
++      g_param_spec_uint ("cores", "Number of logical cores",
++          "Number of logical cores to be used. 0: auto",
++          0, UINT_MAX, PROP_CORES_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_SOCKET,
++      g_param_spec_int ("socket", "Target socket",
++          "Target socket to run on. -1: all available",
++          -1, 1, PROP_SOCKET_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_TILE_ROW,
++      g_param_spec_uint ("tile-row", "Tile Row Count",
++          "Tile count in the Row",
++          1, 16, PROP_TILE_ROW_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  g_object_class_install_property (gobject_class, PROP_TILE_COL,
++      g_param_spec_uint ("tile-col", "Tile Column Count",
++          "Tile count in the Column",
++          1, 16, PROP_TILE_COL_DEFAULT,
++          G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
++
++  gst_element_class_set_static_metadata (element_class,
++      "svthevcenc", "Codec/Encoder/Video", "H265 Encoder",
```

Comment:
Could you make "H265 Encoder" to:
"Scalable Video Technology for HEVC Encoder (SVT-HEVC Encoder)"